04_classwork_basic_calculator/index_calculator.html

```diff
@@ -9,35 +9,41 @@
     <div class="main">
         <div class="inputForms">
           <h2>Ultrabasic calculator</h2>
-            <input type="number" id="numberInput" class="numberInput" required="required">
-            <button id="calculateButton" class="calculateButton">Calculate</button>
-            <button class="resetButton" onClick="history.go(0)">Reset</button>
+            <input type="number" id="numberInput" class="numberInput">
+            <button id="calculateBtn" class="calculateButton">Calculate</button>
+            <button id="resetBtn" class="resetButton">Reset</button>
         </div>
         <div class="numbers">
-            <ul id="numberlList">
+            <ul id="numberslList">
             </ul>
             <div class="result">
                 <p id="sumTitle" class="sumTitle"></p>
             </div>
         </div>
     </div>
     <script>
-        const numberListEl = document.getElementById('numberlList');
+        const numbersListEl = document.getElementById('numberslList');
         const numberInputEl = document.getElementById('numberInput');
-        const numberAddBtn = document.getElementById('calculateButton');
-        let sum = 0;
+        const numberAddBtn = document.getElementById('calculateBtn');
+        let  resetBtnEl = document.getElementById('resetBtn');
         let sumEl = document.getElementById("sumTitle");
+        let sum = 0;
+
+        function reset() {
+           window.history.go(0);
+        }
 
         function addNumber() {
             const number = numberInputEl.value;
-            const numberEl = document.createElement('li');
+            const NewNumberEl = document.createElement('li');
             numberEl.textContent = number;
```

Comment:
this code would not work, because you changed the name of element but forgot to change it in where you're using it

Reply:
Fixed